src/calculator/index.ts

```diff
@@ -98,11 +99,11 @@ export type CalculatorOptions = {
   roundId: string;
   minimumAmountUSD?: number;
   matchingCapAmount?: bigint;
-  passportThreshold?: number;
   enablePassport?: boolean;
   ignoreSaturation?: boolean;
   overrides: Overrides;
   chain: Chain;
+  proportionalMatchOptions: ProportionalMatchOptions;
```

Comment:
Everything in CalculatorOptions is already an "option" so there might not be a need to have an Options suffix here, like the other options in it.